R/simulate.r

```diff
@@ -58,5 +80,24 @@ simulate_mssql <- function() {
 
 }
 
+#' @export
+#' @rdname tbl_lazy
+simulate_hive <- function() {
+  structure(
+    list(),
+    class = c("Hive", "DBITestConnection")
```

Comment:
If I use `DBIConnection` I get:  `unable to find an inherited method for function ‘dbQuoteIdentifier’ for signature ‘"Impala", "ident"...`, but if I use `DBITestConnection` and add `sql_select` to it, all tests work.  I figured I add the method where its least impactful